### DISCUSSION TITLE
Switching from 'Bandage' to 'BandageNG' in homebrew-cask

### DISCUSSION THREAD

**Discussion by kojix2** [Homebrew/discussions#4848] [Casks]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Hi

Right now, homebrew-cask has a software called [Bandage](https://github.com/rrwick/Bandage). It's used to see genome graphs. But these days, more people are using another version called [BandageNG](https://github.com/asl/BandageNG).

So, I'm thinking we should change it. Maybe we can remove "Bandage" and add "BandageNG" instead.

https://github.com/Homebrew/homebrew-cask/blob/master/Casks/b/bandage.rb

### Relevant casks

bandage

**Comment by kojix2** [comment]
### [Bandage](https://github.com/rrwick/Bandage/graphs/contributors)

![image](https://github.com/Homebrew/discussions/assets/5798442/5a8070f4-21a3-40b5-9aa1-f3b5e05e6372)

### [BandageNG](https://github.com/asl/BandageNG/graphs/contributors)

![image](https://github.com/Homebrew/discussions/assets/5798442/c97f44a8-a0cb-4ace-a223-87e11d98488e)

**Reply by SMillerDev** [reply]
https://docs.brew.sh/Adding-Software-to-Homebrew should help you add BandageNG

**Reply by kojix2** [reply]
Thanks.
The decision to add a new package may be easy, but I think the decision to remove an old Formula is difficult. Especially in areas where there are few users.

**Reply by cho-m** [reply]
One option is to add BandageNG and add a caveat to Bandage recommending it. Not sure if we can claim Bandage is [`discontinued`](https://docs.brew.sh/Cask-Cookbook#caveats-mini-dsl) yet based on info at https://github.com/rrwick/Bandage#2022-update but upstream does mention BandageNG as most active fork.
